### PR TITLE
Update and rename full-stack-engineer.md to Lead Backend Developer.md

### PR DESCRIPTION
**Pull request by nwarder**

### CODE REVIEW

File: Lead Backend Developer.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+This is a high-impact, high-opportunity technical role. You'll be working with modern technologies to build a product that has a high throughput. You'll be talking with actual customers to create features that make a real difference.
+
+<!-- more -->
+
+__What we offer__
+
+* Competitive salary + equity + benefits + vacation + parental leave
+* Be challenged and rewarded as you take a lead in our back end development
+* Rapid professional growth
+* Work with an amazing team of like minded professionals/ tech nerds
```
Comment (demoore) [inline Lead Backend Developer.md]: professionals/ tech -> professionals/nerds

File: Lead Backend Developer.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+* Fun team culture - annual team retreat, group lunches, birthday fest
```
Comment (demoore) [inline Lead Backend Developer.md]: birthday fests*

File: Lead Backend Developer.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+* Fun team culture - annual team retreat, group lunches, birthday fest 
+* Unique experiences working with our customer base to solve problems
+* Collaborative approach to solving problems and creating new features
+
+
+
+__Who are you?__
+
+* You have 2+ years of experience building web applications.
+* You ship code quickly; we deploy several times a day.
```
Comment (demoore) [inline Lead Backend Developer.md]: what about `You ship code quickly; we deploy frequently.`

File: Lead Backend Developer.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+* You're passionate about startups and want to join a small company trying to take over the world.
+
+
+
+__What will you be doing?__
+
+* Working with our engineering team to design, build and scale platform features.
+* Making high-level technical decisions about architecture and new technologies.
+* Identifying performance problems and crafting creative solutions.
+* Experimenting with new technologies and solutions
```
Comment (demoore) [inline Lead Backend Developer.md]: why u no full stop? (.)

File: Lead Backend Developer.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+* Separating our primary app into separate microservices.
+* Interacting with customers to learn common road-blocks, bottlenecks and technical requirements.
+
+
+
+__Our Stack__
+
+Our primary application is built using Python and Django, and is hosted on Heroku. We use Postgres, Redis, and DynamoDB to store most of our data, and prefer using hosted services whenever feasible.
+
+We've begun moving smaller systems into separate microservices, written in Go. Our team is committed to using Go for more system tasks as we continue to scale.
```
Comment (demoore) [inline Lead Backend Developer.md:62]: comma prolly unnecessary

File: Lead Backend Developer.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+Sendwithus is a YC W2014 backed company looking for an amazing Lead Backend Developer. We are a fun loving, hard working, and motivated team that loves to ship code and create amazing customer experiences. We want someone who can work with our team of developers and contribute to our lively team culture. We are a small start-up based in Victoria BC and San Francisco CA and we want that perfect person to grow and scale with us.
+
+<!-- more -->
+
+__DESCRIPTION__
+
+We're looking for a full stack engineer to help build new application features, scale our backend systems, and grow into a senior leader on our engineering team.
+As an early engineer, you'll be joining a rapidly growing startup where you will be a leader in scaling a product people love while building a billion dollar company.
+This is a high-impact, high-opportunity technical role. You'll be working with modern technologies to build a product that has a high throughput. You'll be talking with actual customers to create features that make a real difference.
```
Comment (demoore) [inline Lead Backend Developer.md]: You'll be working with modern technologies to build a complex high throughput product.

File: Lead Backend Developer.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
+---
+---
+
+# Senior Backend Developer
+
+_Victoria BC &middot; Full-Time_
+
+Sendwithus is a YC W2014 backed company looking for an amazing Lead Backend Developer. We are a fun loving, hard working, and motivated team that loves to ship code and create amazing customer experiences. We want someone who can work with our team of developers and contribute to our lively team culture. We are a small start-up based in Victoria BC and San Francisco CA and we want that perfect person to grow and scale with us.
```
Comment (demoore) [inline Lead Backend Developer.md]: scale with us -> scalewithus®

Comment (brandonb927) [inline Lead Backend Developer.md]: scalewithus™

**Pull request merged**